scripts/auto_plot.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#Run this script from a directory with an RTS solution and it will create the analyasis plots for you
+
+ATTEMPT_NUM=$(find . -mindepth 1 -maxdepth 1 -type d | wc -l)
+ATTEMPT_NUM=$(expr $ATTEMPT_NUM + 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4498157)